docker-compose-prod.yml

```diff
@@ -11,6 +11,7 @@ services:
       - PORT=4000
       - NODE_ENV=production
     env_file: .env
+    command: sh -c './node_modules/.bin/wait-port ccm_db_prod:3306 && ./start.sh'
```

Comment:
This looks pretty good! I wish we could simplify the `wait-port` command, but seems that's the documented approach. I know I suggested it, but the `start.sh` file feels borderline unnecessary (the alternative being sticking to `npm` commands). However, I'm imagining that we may want to additional commands for prod at some point, so I'm good with this. The ability to do some logging is also nice.

Reply:
Yeah.. I like putting the additional logic in `start.sh` which we don't have now, but I time open up this would be nice [improvement](https://github.com/tl-its-umich-edu/canvas-course-manager-next/issues/131)